### PR TITLE
Introduce material ui

### PR DESCRIPTION
**Pull request by castrolem**
**What:**
Introduce Material UI for Admin views to improve the visualization of the data.

**Why:**
To improve UI friendliness to the end-user.

**How:**
Using React and Material UI packages, to create an admin 
image

### CODE REVIEW

File: app/javascript/bundles/Admin/components/Subscriptions/Table.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import numeral from 'numeral'
+import { makeStyles } from '@material-ui/core/styles'
+import Table from '@material-ui/core/Table'
```
Comment (orlando) [inline app/javascript/bundles/Admin/components/Subscriptions/Table.jsx:5]: @castrolem can we import multiple components with just one import syntax? Something like

```js
import { Table, TableRow ... } from "@material-ui"
```

File: app/views/admin/plans/index.html.erb [hunk 2]
```diff
@@ -2,26 +2,7 @@
...
-        <td><%= plan.description %></td>
-        <td>$<%= plan.amount %></td>
-        <td><%= link_to 'Show', admin_plan_path(plan) %></td>
-        <td><%= link_to 'Edit', edit_admin_plan_path(plan) %></td>
-        <td><%= link_to 'Destroy', plan, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<%= react_component("PlansTable", props: {plans: @plans}, prerender: false) %>
```
Comment (orlando) [inline app/views/admin/plans/index.html.erb:5]: @castrolem what's the reason behind `prerender: false`?

**Review by orlando — APPROVED** [review]
A few comments but LGTM

File: app/views/admin/plans/index.html.erb (hunk 2 above)
Comment (castrolem) [reply app/views/admin/plans/index.html.erb:5]: According to the docs: In most cases, you should use the prerender: false (default behavior) with the provided helper method to render the React component from your Rails views.

File: app/javascript/bundles/Admin/components/Subscriptions/Table.jsx (hunk 1 above)
Comment (castrolem) [reply app/javascript/bundles/Admin/components/Subscriptions/Table.jsx:5]: We could, to me it looked better and more explicit this way, but that's just preference.

**Pull request merged**